### PR TITLE
deps(puppeteer): upgrade to 13.7.0

### PR DESCRIPTION
**Pull request by connorjclark**

### CODE REVIEW

**Comment by connorjclark** [comment]
Very odd that `report-renderer-axe-test.js` is failing just because puppeteer/Chrome updated.


1) duplicate ids
```
Document has multiple static elements with the same id attribute: viewport, document-title, etc. ...
```

2) color contrast
```
Element has insufficient color contrast of 4.37 (foreground color: #757575, background color: #f8f9fa, font size: 9.0pt (12px), font weight: normal). Expected contrast ratio of 4.5:1",
    +             "relatedNodes": Array [
    +               Object {
    +                 "html": "<tr class=\"lh-row--even\">",
    +                 "target": Array [
    +                   ".lh-audit--fail.lh-audit--binary.lh-audit:nth-child(3) > .lh-expandable-details > table > tbody > .lh-row--even:nth-child(9)",
    +                 ],
    +               },
    +               Object {
    +                 "html": "<article class=\"lh-root lh-vars lh-screenshot-overlay--enabled\" style=\"--element-screenshot-url:url(&quot;data:image/jpeg;base64,/9j/4AAQSkZJRgABAQAAAQABAAD/2wBDABsSFBcUERsXFhceHBsgKEIrKCUlKFE6PTBCYFVlZF9VXVtqeJmBanGQc1tdhbWGkJ
```

File: report/test/renderer/report-renderer-axe-test.js [hunk 1]
```diff
@@ -53,23 +53,351 @@ describe('ReportRendererAxe', () => {
...
+        ],
+      });
+
+      const axeSummary = axeResults.violations.map((v) => {
+        return {
+          id: v.id,
+          message: v.nodes.map((n) => n.failureSummary).join('\n'),
+        };
+      });
+      expect(axeSummary).toMatchInlineSnapshot(`
```
Comment (adamraine) [inline report/test/renderer/report-renderer-axe-test.js:79]: This is kinda long, and it's easy to miss the next expectation. Can you stick this in the snap file as well?

Comment (connorjclark) [reply report/test/renderer/report-renderer-axe-test.js:79]: I plan to resolve the color-contrast issues before merging.

**Comment by connorjclark** [comment]
GH won't pick up the latest branch's commit in this PR...

**Comment by connorjclark** [comment]
Splitting out dealing with the color contrast changes. #13918

**Comment by adamraine** [comment]
Two network timeouts on yarn install. Could that be a coincidence or is puppeteer getting bigger?

**Review by adamraine — APPROVED** [review]

**Pull request merged**